.github/workflows/test-all-packages.yml

```diff
@@ -266,10 +266,10 @@ jobs:
       if: steps.built.outputs.cache-hit != 'true'
     # END-RESTORE-BOILERPLATE
 
-    - name: Set up Go 1.14
+    - name: Set up Go 1.15
       uses: actions/setup-go@v1
       with:
-        go-version: 1.14
+        go-version: 1.15
```

Comment:
https://github.com/Agoric/agoric-sdk/blob/master/packages/cosmic-swingset/README.md still says 1.13

Seems like we document the go version somewhere else too... ah... I see you just updated https://github.com/Agoric/agoric-sdk/wiki/Validator-Guide to say 1.15

Reply:
I'll update the other places, too...

Reply:
Done.